### ISSUE TITLE
Compiled executable

### ISSUE THREAD

**Issue by joseaugustin** [manutalcual/winredock#13]
Hi Manuel -

Non developer here. WinReDock seems exactly what I am looking for. Is there a compiled executable available for download for Win10? I am not a developer and compiling is not my cup of coffee!

Jose

**Comment by manutalcual** [comment]
Yep! Open the  I0010-restoring-positions-doesnt-work branch and in the "bin" directory there are two zip files. Try first the non-loggin version (that is bin.zip) You can unzip that on a folder and give it a try.
If you have problems with this I can attach a copy here.

How have you found this? Through superuser?

Manuel

**Comment by joseaugustin** [comment]
Thanks Manuel. Yes. I found you on SuperUser. I was able to download the non-logging version and get it extracted. However it doesn't seem to be working.

I did Get Windows --> Save Config --> Moved the applications over to other screens --> Read config. On read config, nothing happened. Where is the JSON file created? I did not see a JSON file in the application folder.

**Comment by manutalcual** [comment]
You don't need to read the config. There are changes to that automatically.
If you position your windows, undock the laptop and re-dock it, they should go to their position automatically. (You have to wait one second between undock and redock.)
If you need to save the windows positions manually, then download the "master" branch (bin dir) and do a read/save menu operation as explained in the master README.
If they don't there is another bug.
If you like the app (if it works for you), could you upvote it in superuser?

Thanks.

**Comment by joseaugustin** [comment]
Doesn't work for me. Must be a bug that I am seeing. I tried both manual position save and automatic - the windows do not re-position on redock. Is there any information I can provide you to troubleshoot?

**Comment by manutalcual** [comment]
Yes. Try the loggin version and attach here the log. The log is wm.log and should be in the same directory you start the app on.

Thank you very much for your interest and help.

**Comment by joseaugustin** [comment]
Here you go.
[wm.log](https://github.com/manutalcual/winredock/files/2725070/wm.log)

**Comment by manutalcual** [comment]
Ok. I've removed something. I've tested and it works.
Try again and let me know if you have problems.

**Comment by slundy** [comment]
I assume this would work for 'tv monitors' that are connected to a pc.  We have two 65" tv's with various telemetry and monitoring data on them and EVERY morning I have to remote in, move one window over to the other screen.  If I could get it to stick to one on each screen I'd be a much happier tech.

**Comment by bittner** [comment]
Just wanted to say a great big **"Thank you"** for this awesome piece of software that automates some daily pain away! 🥇
 
(Finally, my workplace machine behaves more like my home development box with Ubuntu's GNOME desktop.) 🐧 😬 

I simply took the [bin.zip](https://github.com/manutalcual/winredock/tree/master/bin) from the `bin/` folder and followed the [installation instructions](https://github.com/manutalcual/winredock#installation) in the README. 🏁 
I have two external monitors attached via a Lenovo docking station, attached to the laptop via USB-C.
